tests/queries/test_query.py

```diff
@@ -140,21 +158,33 @@ def test_filter_conditional(self):
 
     def test_filter_conditional_join(self):
         query = Query(Item)
-        filter_expr = Func('note__note', output_field=BooleanField())
-        msg = 'Joined field references are not permitted in this query'
+        filter_expr = Func("note__note", output_field=BooleanField())
+        msg = "Joined field references are not permitted in this query"
         with self.assertRaisesMessage(FieldError, msg):
             query.build_where(filter_expr)
 
     def test_filter_non_conditional(self):
         query = Query(Item)
-        msg = 'Cannot filter against a non-conditional expression.'
+        msg = "Cannot filter against a non-conditional expression."
         with self.assertRaisesMessage(TypeError, msg):
             query.build_where(Func(output_field=CharField()))
 
+    @mock.patch("django.db.models.sql.query.connections")
+    def test_sql_with_params_using_non_default_db(self, connections):
+        query = Query(Item, using=OTHER_DATABASE_ALIAS)
+
+        query.sql_with_params()
+
+        # We cannot update DATABASES in settings.py for a single test to remove
+        # the default DB to test that the method call works. We can fall back to
+        # checking that the correct connection is looked up and assume that it's
+        # used.
+        connections.__getitem__.assert_called_once_with(OTHER_DATABASE_ALIAS)
+
 
 class JoinPromoterTest(SimpleTestCase):
     def test_repr(self):
         self.assertEqual(
-            repr(JoinPromoter('AND', 3, True)),
```

Comment:
Please revert all unrelated changes from single to double quotes.

Reply:
I can do this. For future reference, is this to keep the change specific, or is there a preference for single quotes?

Reply:
Currently both.

Reply:
Cool, got it. Let me update the code I've written to use single quotes too then.